### PR TITLE
Connect all page.

### PR DESCRIPTION
**Pull request by Mayankrawat01**

### CODE REVIEW

File: createAcc.html [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="./signup.css">
+</head>
+<body>
+    <div class='loginsignup'>
+        <div class="loginsignup-container">
+          <h1>Create Account</h1>
+          <div class="loginsignup-fields">
```
Comment (baibhav-celestial) [inline createAcc.html:13]: Why there is no validation?

File: createAcc.html [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+            <input type="password" placeholder='password' id="password"/>
+
+          </div>
+          <div class="loginsignup-agree">
+            <input type="checkbox" id="alert" />
+<label for="alert">I accept the terms and conditions</label>
+          </div>
+          <button type="button" onclick="createaccount()" >Create Account</button>
+          <a href="./SignupPage.html"><p>Login </p></a>
+
```
Comment (baibhav-celestial) [inline createAcc.html:30]: Why there is no styling?

**Comment by Mayankrawat01** [comment]
yes sir

File: createAcc.html (hunk 1 above)
Comment (Mayankrawat01) [reply createAcc.html:13]: I added the validation in it now

File: createAcc.html (hunk 2 above)
Comment (Mayankrawat01) [reply createAcc.html:30]: I also the styling in it

**Pull request merged**